index.js

```diff
@@ -75,6 +77,7 @@ const packages = {
   'axboot': 'https://opencollective.com/ax-boot-framework',
   'babel': 'https://opencollective.com/babel',
   'babel-core': 'https://opencollective.com/babel',
+  'babel-preset-cgb': 'https://pay.paddle.com/checkout/515568',
```

Comment:
Please remove all the individual packages from this section. It's sufficient to add yourself to the `authors` object for all your packages to be detected.